spec/lib/rbac/filterer_spec.rb

```diff
@@ -796,12 +796,18 @@ def get_rbac_results_for_and_expect_objects(klass, expected_objects)
       describe ".search" do
         let!(:ansible_configuration_script)          { FactoryGirl.create(:ansible_configuration_script) }
         let!(:ansible_configuration_script_with_tag) { FactoryGirl.create(:ansible_configuration_script) }
+        let!(:ansible_playbook)                      { FactoryGirl.create(:ansible_playbook) }
+        let!(:ansible_playbook_with_tag)             { FactoryGirl.create(:ansible_playbook) }
 
         it 'works when targets are empty' do
           User.with_user(user) do
             results = described_class.search(:class => 'ConfigurationScript').first
             expect(results.length).to eq(2)
             expect(results).to match_array([ansible_configuration_script, ansible_configuration_script_with_tag])
+
+            results = described_class.search(:class => 'ConfigurationScriptPayload').first
+            expect(results.length).to eq(2)
```

Comment:
`match_array` already covers the length of the array.  This expectation isn't needed.